lib/gui/components/update-notifier/services/update-notifier.js

```diff
@@ -16,13 +16,16 @@
 
 'use strict';
 
+const _ = require('lodash');
 const semver = require('semver');
 const etcherLatestVersion = require('etcher-latest-version');
 
 module.exports = function($http, $q, ModalService, UPDATE_NOTIFIER_SLEEP_TIME, ManifestBindService, SettingsModel) {
 
   /**
-   * @summary Get the latest available Etcher version
+   * @summary Get the latest available Etcher version and memoize it;
```

Comment:
Ah, lets also keep the summary as it is now, and use `@description` for the rest:

```
@summary Get the latest available Etcher version
@description
foo bar
```